### ISSUE TITLE
convert conding from ISO-8859-1  to utf-8?

### ISSUE THREAD

**Issue by shouhuanxiaoji** [fofix/fofix#57]
ISO-8859-1 can only support Latin characters but not east-Asian characters.
So after I translate the game, all characters is garbled.

**Comment by L0FKA** [comment]
"translate the game" you mean code encoding? or strings in utf-8? I've done localization for fofix 3.x back in days, still have repo here, never tested dev branch of it, just v4a2 and it's looks nice, but still missing some strings to translate

**Comment by shouhuanxiaoji** [comment]
I means this translate——create a new .po file from .pot, them translate it. 
then compile it to .mo, them let game to load the new language. 
I want to translate the latest version 4.0.0alpha to Chinese

**Comment by L0FKA** [comment]
You need to be sure that game works with strings as unicode or uses utf-8 then all you need is font that supports your language glyphs and do actual translation then.

**Comment by shouhuanxiaoji** [comment]
I have change all font files of game to font that support Chinese.

**Comment by L0FKA** [comment]
have you tried to play rocksmith 2014? It has Japanese but I've got Chinese translation, you probably could find it easily :D

just real instrument and decent game as for me.

no ads just asking, I migrated over to that kind of games from gh\fof\ps games. :P

**Comment by shouhuanxiaoji** [comment]
rocksmith does not supoort linux...

**Comment by L0FKA** [comment]
that's issue, dunno if mac version easy to port to linux, but that would be not that legal, haha. point catched.

**Comment by shouhuanxiaoji** [comment]
hm...

**Comment by mdsitton** [comment]
Speaking of rocksmith, I actually recently bought a guitar and that game should get here in the next couple of days lol(would be interesting to adapt the vocals pitch detection to do guitar stuff but probably not gonna happen). 

Regarding this issue, I'd need to verify that FoFiX can properly display Muti Byte Unicode characters, because i'm fairly sure that FoFiX's font rendering was never setup for this kind of stuff. Its fairly basic as far as i know, and i need to rewrite it to be able to move off from pygame. But this guess could be wrong who knows...

**Comment by shouhuanxiaoji** [comment]
if you have a result. Please tell me.
I will pull request my translation.

**Comment by stump** [comment]
Does the .po file specify the correct encoding in its Content-Type header?

The function the game uses to get translated strings should obey that header.

**Comment by shouhuanxiaoji** [comment]
yes, I confirm that I have use the correct encoding, because utf-8 is the defalut encode of Linux.If encoding of file is another, editor will disapperas warning.
